modelsim_run/work/_info

```diff
@@ -10,45 +10,45 @@ cModel Technology
 Z0 dC:/Users/gilya/OneDrive/Desktop/studies/4th/project/FinalProjectWS/rvc_asap/modelsim_run
 vrvc_asap
 Z1 DXx6 sv_std 3 std 0 22 VYECXdT12H8WgbUP_5Y6:3
-Z2 DXx4 work 12 rvc_asap_pkg 0 22 OHOhSK@HQ:AT48egM@m^B3
-Z3 !s110 1639590809
+Z2 DXx4 work 12 rvc_asap_pkg 0 22 ?Z_PPeMaSfI2Q7UjAX[:z2
```

Comment:
Please dont Commit this file.
Add to git ignore

Reply:
This whole folder in the .gitignore, I tried but can not get rid of this file.